client/components/payment-logo/docs/example.jsx

```diff
@@ -5,21 +5,45 @@
  */
 
 import React from 'react';
+import concat from 'lodash/fp/concat';
```

Comment:
To shorten this, we could do:

`import { concat, filter, flow, map, sortBy } from 'lodash';`

:)

Reply:
Hi @bluefuton, I’m importing with this verbosity here is because I need auto-curried iteratee-first data-last methods (those provided by `lodash/fp`) to pass into `flow`. And to use `lodash/fp`, it seems to me that we have to import each method line by line. `import { concat, filter, flow, map, sortBy } from 'lodash/fp';` is not valid.